src/test/rustdoc/glob-shadowing.rs

```diff
@@ -0,0 +1,38 @@
+// @has 'glob_shadowing/index.html'
+// @count - '//tr[@class="module-item"]' 2
```

Comment:
Can you also test there's only one function named "describe"?

Reply:
I see you added a test `sub2::describe` exists - could you also check no other describe functions exist? Maybe something like `@!has - '//tr[@class="module-item"]' 'sub1::describe'`